k8s.gcr.io/images/k8s-staging-jobset/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS file documentation:
```

Comment:
it seems that this now should be under "k8s.io/registry.k8s.io/images"

Reply:
Deleted these and recreated them in `k8s.io/registry.k8s.io/images/k8s-staging/jobset`